py3status/modules/external_script.py

```diff
@@ -29,8 +29,7 @@
 
 @author frimdo ztracenastopa@centrum.cz
 """
-
```

Comment:
Involuntary manslaughter. I engaged in reckless behavior weeks ago. :-(

I tested this module today. I think it shouldn't go in.

Sometimes I receive exceptions due to the nature of scripting... `grep` picked up nothing this time... or `curl` when internet is down... etc can return error codes. `1`,`2`, etc...

I'm revisiting this again because I hadn't experienced any `xsel` / `error 15` fiasco thanks to you. Also, we could use external.py as a drop-in replacement w/ bonus features if this PR does not work out.